tools/cpp/windows_cc_configure.bzl

```diff
@@ -300,6 +333,12 @@ def _get_vc_full_version(repository_ctx, vc_path):
         return repository_ctx.os.environ["BAZEL_VC_FULL_VERSION"]
     return _get_latest_subversion(repository_ctx, vc_path)
 
+def _get_winsdk_full_version(repository_ctx):
+    """Return the value of BAZEL_WINSDK_FULL_VERSION if defined, otherwise an empty string."""
+    if "BAZEL_WINSDK_FULL_VERSION" in repository_ctx.os.environ:
```

Comment:
Can you replace this whole function with just this? :

    return repository_ctx.os.environ.get("BAZEL_WINSDK_FULL_VERSION", default = "")

Reply:
Done, thanks!